src/components/pages/dashboard-pages/LessonForm.jsx

```diff
@@ -0,0 +1,165 @@
+            Select a Class
+          </label>
+          <select
+            id="classSelect"
+            name="classId"
+            value={formData.classId}
+            onChange={handleChange}
+            className="mt-4 mb-2 p-2 w-full text-sm placeholder:text-darkGray text-black bg-pureWhite rounded border-2 border-grey appearance-none focus:outline-none focus:ring-0 focus:border-black"
+          >
+            <option value="">Select a class</option>
+            {filteredClasses.map((classItem) => (
+              <option key={classItem._id} value={classItem._id}>
+                {classItem.classTitle}
+              </option>
+            ))}
+          </select>
+          {formErrors.classId && (
+            <p className="text-red text-sm font-spartan">
+              {formErrors.classId}
+            </p>
+          )}
+          <FormInput
+            placeholder=" "
+            name="lessonTitle"
+            value={formData.lessonTitle}
+            onChange={handleChange}
+            min="2"
+            max="100"
+          >
+            Lesson Title
+          </FormInput>
+          {formErrors.lessonTitle && (
+            <p className="text-red text-sm font-spartan">
+              {formErrors.lessonTitle}
+            </p>
+          )}
+          <div className="flex gap-4 items-center">
+            <p className="w-1/4">Date & Time:</p>
+            <div className="flex gap-4 w-3/4">
+              <FormInput
+                type="date"
+                name="date"
+                value={formData.lessonSchedule.date}
+                onChange={handleChange}
+                min={new Date().toISOString().split('T')[0]}
+                className="w-1/2"
+              />
+              <FormInput
+                type="time"
+                name="startTime"
+                value={formData.lessonSchedule.startTime}
+                onChange={handleChange}
+                className="w-1/2"
+              />
+            </div>
+          </div>
+          {formErrors['lessonSchedule.date'] && (
+            <p className="text-red text-sm font-spartan">
+              {formErrors['lessonSchedule.date']}
+            </p>
+          )}
+          {formErrors['lessonSchedule.startTime'] && (
+            <p className="text-red text-sm font-spartan">
+              {formErrors['lessonSchedule.startTime']}
+            </p>
+          )}
+          <div className="flex gap-4 my-4">
+            <p className="w-1/4">Type:</p>
+            <div className="flex flex-col gap-2 w-3/4">
+              <label className="flex gap-1 items-center">
+                <input
+                  type="radio"
+                  name="type"
+                  value="online"
+                  className="w-4 h-4 accent-lightGreen focus:darkGreen"
+                  onChange={handleChange}
+                />
+                Online
+              </label>
+              <label className="flex gap-1 items-center">
```

Comment:
label is missing a for attribute to associate it with its corresponding input.

Reply:
Fixed